common/src/test/java/com/sequenceiq/cloudbreak/util/DatabaseCommonTest.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.cloudbreak.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class DatabaseCommonTest {
+    private static final String ORACLE_URL = "jdbc:oracle:@test.eu-west-1.rds.amazonaws.com:1521/hivedb";
```

Comment:
Suggestion: Use DNS names guaranteed not to resolve anywhere, just in case something in Java decides to be clever.

Reply:
I'm going to push back on this one because there's no network connectivity involved here.